src/test/java/uk/gov/companieshouse/web/accounts/transformer/smallfull/BalanceSheetTransformerTests.java

```diff
@@ -13,7 +13,7 @@
 
 public class BalanceSheetTransformerTests {
 
-    private Integer CALLED_UP_SHARE_CAPITAL = 1;
+    private Long CALLED_UP_SHARE_CAPITAL = Long.valueOf("1");
```

Comment:
For reference, this could be written as `private Long CALLED_UP_SHARE_CAPITAL = 1L;`.